### PR TITLE
example code to manage cronjobs

### PR DESCRIPTION
**Pull request by swwolf**

### CODE REVIEW

File: modules/profiles/manifests/cron.pp [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# gets $data from hiera
+class profiles::cron($data = '') {
```
Comment (ChairmanTubeAmp) [inline modules/profiles/manifests/cron.pp]: Should get all data from params.pp.

**Review by ChairmanTubeAmp — CHANGES_REQUESTED** [review]

File: modules/profiles/manifests/cron.pp [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  package { ['gcc']:
+    ensure  => 'installed',
+  }
+
+  # create cronjobber
+  if (file('/usr/local/bin/cronjobber', '/dev/null') == '') {
+    file { '/tmp/cronjobber.c':
+      ensure  => 'present',
+      source => 'puppet:///modules/profiles/cron/cronjobber.c',
```
Comment (obi11235) [inline modules/profiles/manifests/cron.pp:16]: line up your =>

**Review by obi11235 — CHANGES_REQUESTED** [review]

File: modules/profiles/manifests/cron.pp [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    }
+  }
+  exec { 'compile_cronjobber':
+    command => '/usr/bin/gcc -Wall /tmp/cronjobber.c -o /usr/local/bin/cronjobber; rm /tmp/cronjobber.c',
+    creates => '/usr/local/bin/cronjobber'
```
Comment (d33psky) [inline modules/profiles/manifests/cron.pp]: ,

**Review by d33psky — COMMENTED** [review]
,

**Review by obi11235 — APPROVED** [review]

**Review by ChairmanTubeAmp — APPROVED** [review]

**Pull request merged**